Parsers/ASimProcessEvent/Parsers/ASimProcessCreateVMwareCarbonBlackCloud.yaml

```diff
@@ -0,0 +1,118 @@
+    https://developer.carbonblack.com/reference/cb-threathunter/latest/event-search-fields/
+Description: |
+  This ASIM parser supports normalizing VMware Carbon Black Cloud logs to the ASIM Process Create normalized schema. VMware Carbon Black Cloud events are captured through VMware Carbon Black Cloud data connector which ingests Carbon Black Audit, Notification and Event data into Microsoft Sentinel through the REST API.
+ParserName: ASimProcessCreateVMwareCarbonBlackCloud
+EquivalentBuiltInParser: _ASim_ProcessEvent_CreateVMwareCarbonBlackCloud
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let EventFieldsLookup = datatable(sensor_action_s: string, DvcAction: string, EventResult: string)[
+      "ACTION_ALLOW",	"Allow", "Success",
+      "ACTION_BLOCK",	"Block", "Failure",
+      "ACTION_TERMINATE",	"Terminate", "Failure",
+      "ACTION_BREAK",	"Break", "Failure",
+      "ACTION_SUSPEND", "Suspend", "Failure",
+      "", "", "Success"
+  ];
+  let parser = (disabled: bool=false) {
+      CarbonBlackEvents_CL
+      | where eventType_s == "endpoint.event.procstart" and isnotempty(childproc_pid_d)
+      | parse process_hash_s with * '["' ActingProcessMD5: string '","' ActingProcessSHA256: string '"]'
+      | parse parent_hash_s with * '["' ParentProcessMD5: string  '","' ParentProcessSHA256: string '"]'
+      | parse childproc_hash_s with * '["' TargetProcessMD5: string '","' TargetProcessSHA256: string '"]'
+      | lookup EventFieldsLookup on sensor_action_s
+      | extend
+          EventStartTime = todatetime(split(createTime_s, '+')[0]),
+          TargetProcessId = tostring(toint(childproc_pid_d)),
+          ActingProcessId = tostring(toint(process_pid_d)),
+          ParentProcessId = tostring(toint(parent_pid_d)),
+          AdditionalFields = bag_pack(
+                      "org_key", org_key_s,
+                      "alert_id", alert_id_g,
+                      "parent_cmdline", parent_cmdline_s,
+                      "process_reputation", process_reputation_s,
+                      "childproc_reputation", childproc_reputation_s,
+                      "parent_reputation", parent_reputation_s,
+                      "process_guid", process_guid_s,
+                      "childproc_guid", childproc_guid_s,
+                      "parent_guid", parent_guid_s
+                  )
+      | invoke _ASIM_ResolveDvcFQDN('device_name_s')
+      | project-rename 
+          ActorUsername = process_username_s,
+          TargetProcessCommandLine = target_cmdline_s,
+          TargetProcessName = childproc_name_s,
+          TargetUsername = childproc_username_s,
+          DvcIpAddr = device_external_ip_s,
+          DvcScope = device_group_s,
+          ActingProcessCommandLine = process_cmdline_s,
+          ActingProcessName = process_path_s,
+          DvcId = device_id_s,
```

Comment:
In our environment, this column is populated as device_id_s in the table with values like 6697319.
![image](https://github.com/Azure/Azure-Sentinel/assets/107753570/2f770c39-9f12-44b8-8ec1-c0f7b560d868)

Reply:
We double checked the sample logs and in that the value of device_id is real instead of string with double quotes.
So updated raw logs. Please try to ingest it again. Thanks